### PR TITLE
Added new Changelogs for v0.102.0

### PR DESCRIPTION
**Pull request by samiura**
**Description:** Changelogs associated with the latest release .

**Link to Splunk idea:** <Link to Splunk idea, see https://ideas.splunk.com>

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,9 +2,51 @@
...
+
+
+### 🧰 Bug fixes 🧰
+
+- (Splunk) `discovery`: Fix crashing collector if discovered mongodb isn't reachable in Kubernetes ([#4911](https://github.com/signalfx/splunk-otel-collector/pull/4911))
+- (Contrib) `podmanreceiver`: add scraper's shutdown method ([#29994](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29994))
+- (Contrib) `awsxrayexporter`: Fix the DB subsegment(client span) name with JDBC conn string starts with "jdbc:" ([#33225](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33225))
+- (Contrib) `exp/metrics`: fixes staleness.Evict such that it only ever evicts actually stale metrics ([#33265](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33265))
+- (Contrib) `receiver/mysql`: Remove the order by clause for the column that does not exist ([#33271](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33271))
+- (Contrib) `influxdb(exporter|receiver)`: remove Metric flags field to/from InfluxDB conversion ([#29896](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29896))
```
Comment (crobert-1) [inline CHANGELOG.md:43]:
```suggestion
```
We don't have influxdb components in our distro.

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,9 +63,39 @@
...
+- (Contrib) `stanza`: Add monitoring metrics for open and harvested files in fileconsumer ([#31256](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31256))
+- (Contrib) `awss3receiver`: Uses obsreport to report metrics for the AWS S3 Receiver. ([#30750](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30750))
+- (Contrib) `awsxrayexporter`: AWS X-Ray exporter to make local root spans a segment for internal/service spans and subsegment + segment for client/producer/consumer spans. ([#33000](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33000))
+- (Contrib) `prometheusreceiver`: Allow to configure http client used by target allocator generated scrape targets ([#18054](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/18054))
+- (Contrib) `clickhouseexporter`: Add `create_schema` option to ClickHouse exporter ([#32282](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32282))
+  The new create_schema option allows disabling default DDL to let the user manage their own schema.
```
Comment (crobert-1) [inline CHANGELOG.md:81]:
```suggestion
```
Component is not in Splunk distro

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,9 +2,51 @@
...
+  For most clients, this shouldn't be a problem, but if you often have payloads that decompress to more than 20 MiB, you might want to either configure your
+  client to send smaller batches (recommended), or increase the limit using the MaxRequestBodySize option.
+
+
+### 🧰 Bug fixes 🧰
+
+- (Splunk) `discovery`: Fix crashing collector if discovered mongodb isn't reachable in Kubernetes ([#4911](https://github.com/signalfx/splunk-otel-collector/pull/4911))
+- (Contrib) `podmanreceiver`: add scraper's shutdown method ([#29994](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29994))
+- (Contrib) `awsxrayexporter`: Fix the DB subsegment(client span) name with JDBC conn string starts with "jdbc:" ([#33225](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33225))
+- (Contrib) `exp/metrics`: fixes staleness.Evict such that it only ever evicts actually stale metrics ([#33265](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33265))
```
Comment (crobert-1) [inline CHANGELOG.md:41]:
```suggestion
```
I believe this package is only used by the `deltatocumulative` processor, which isn't in our distro, so this does not effect us.

File: CHANGELOG.md [hunk 4]
```diff
@@ -21,9 +63,39 @@
...
 
-### 🧰 Bug fixes 🧰
+### 💡 Enhancements 💡
+
+- (Splunk) `discovery`: Update redis discovery instructions ([#4915](https://github.com/signalfx/splunk-otel-collector/pull/4915))
+- (Splunk) `discovery`: Bring Kafkamatrics receiver into the discovery mode ([#4903](https://github.com/signalfx/splunk-otel-collector/pull/4903))
+- (Contrib) `pkg/ottl`: Add the `Day` Converter to extract the int Day component from a time.Time ([#33106](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33106))
+- (Contrib) `pkg/ottl`: Adds `Month` converter to extract the int Month component from a time.Time (#33106) ([#33106](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33106))
+- (Contrib) `cmd/telemetrygen`: Add support for adding spanID and traceID as exemplars to datapoints generated by telemetrygen ([#33320](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33320))
+- (Contrib) `cmd/telemetrygen`: Add support for specifying trace ID and span ID in telemetrygen for logs ([#33234](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33234))
```
Comment (crobert-1) [inline CHANGELOG.md:73]: Our only dependency is in some of our examples, I'm not sure whether or not these should be included. Maybe someone else has opinions here?

File: CHANGELOG.md [hunk 5]
```diff
@@ -21,9 +63,39 @@
...
+- (Contrib) `pkg/ottl`: Adds a `Year` converter for extracting the int year component from a time.Time ([#33106](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33106))
+- (Contrib) `filelogreceiver`: Log when files are rotated/moved/truncated ([#33237](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33237))
+- (Contrib) `stanza`: Add monitoring metrics for open and harvested files in fileconsumer ([#31256](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31256))
+- (Contrib) `awss3receiver`: Uses obsreport to report metrics for the AWS S3 Receiver. ([#30750](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30750))
```
Comment (crobert-1) [inline CHANGELOG.md:77]:
```suggestion
```
Component is not in our distro

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,9 +2,51 @@
...
 - (Splunk) `receiver/discovery`: Replace `log_record` field with `message` in evaluation statements ([#4583](https://github.com/signalfx/splunk-otel-collector/pull/4583))
+- (Contrib) `k8sattributesprocessor`: Move `k8sattr.rfc3339` feature gate to stable. ([#33304](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33304))
+- (Contrib) `extension/opamp`: Redact all values in the effective config ([#33267](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33267))
+  All values will be treated as if they are a `configopaque.String` type. This will
+  be changed once the Collector APIs are updated to unmarshal the config while
+  only redacting actual `configopaque.String`-typed values.
+
+  The exception to redaction is the `service::pipelines` section, which is useful
+  for debugging and does not contain any `configopaque.String` values.
+
```
Comment (crobert-1) [inline CHANGELOG.md:18]:
```suggestion
```
Not in our distro

File: CHANGELOG.md (hunk 4 above)
Comment (samiura) [reply CHANGELOG.md:74]: @pjanotti what do you think?

File: CHANGELOG.md (hunk 5 above)
Comment (samiura) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 7]
```diff
@@ -2,9 +2,51 @@
...
+  This change applies the same restriction to the decompressed content. As a security measure, a limit of 20 MiB was added, which makes this a breaking change. 
+  For most clients, this shouldn't be a problem, but if you often have payloads that decompress to more than 20 MiB, you might want to either configure your
+  client to send smaller batches (recommended), or increase the limit using the MaxRequestBodySize option.
+
+
+### 🧰 Bug fixes 🧰
+
+- (Splunk) `discovery`: Fix crashing collector if discovered mongodb isn't reachable in Kubernetes ([#4911](https://github.com/signalfx/splunk-otel-collector/pull/4911))
+- (Contrib) `podmanreceiver`: add scraper's shutdown method ([#29994](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29994))
+- (Contrib) `awsxrayexporter`: Fix the DB subsegment(client span) name with JDBC conn string starts with "jdbc:" ([#33225](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33225))
```
[the remark this answers is not in the document]
Comment (samiura) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 1 above)
Comment (samiura) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 8]
```diff
@@ -2,9 +2,51 @@
...
+- (Contrib) `exp/metrics`: fixes staleness.Evict such that it only ever evicts actually stale metrics ([#33265](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33265))
+- (Contrib) `receiver/mysql`: Remove the order by clause for the column that does not exist ([#33271](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33271))
+- (Contrib) `influxdb(exporter|receiver)`: remove Metric flags field to/from InfluxDB conversion ([#29896](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29896))
+- (Contrib) `kafkareceiver`: Fix bug that was blocking shutdown ([#30789](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30789))
+- (Contrib) `exporter/datadog`: Fixes a potential race condition when the traces exporter and metrics exporter are both shutting down. ([#33291](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33291))
```
[the remark this answers is not in the document]
Comment (samiura) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 9]
```diff
@@ -21,9 +63,39 @@
...
+- (Core) `mdatagen`: support setting an AttributeSet for async instruments [#9674](https://github.com/open-telemetry/opentelemetry-collector/issues/9674)
+- (Core) `mdatagen`: support using telemetry level in telemetry builder [#10234](https://github.com/open-telemetry/opentelemetry-collector/issues/10234)
+  This allows components to set the minimum level needed for them to produce telemetry. By default, this is set to configtelemetry.LevelBasic. If the telemetry level is below that minimum level, then the noop meter is used for metrics.
+- (Core) `mdatagen`: add support for bucket boundaries for histograms [#10218](https://github.com/open-telemetry/opentelemetry-collector/issues/10218)
+- (Core) `releases`: add documentation in how to verify the image signatures using cosign [#9610](https://github.com/open-telemetry/opentelemetry-collector/issues/9610)
+
+### 🚀 New components 🚀
+
+- (Contrib) `geoipprocessor`: introduce the GeoIP processor ([#32663](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/32663))
```
[the remark this answers is not in the document]
Comment (samiura) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 3 above)
Comment (samiura) [reply CHANGELOG.md]: removed

File: CHANGELOG.md (hunk 6 above)
Comment (samiura) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 10]
```diff
@@ -21,9 +63,39 @@
...
+- (Contrib) `cmd/telemetrygen`: Add support for specifying trace ID and span ID in telemetrygen for logs ([#33234](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33234))
+- (Contrib) `pkg/ottl`: Adds a `Year` converter for extracting the int year component from a time.Time ([#33106](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33106))
+- (Contrib) `filelogreceiver`: Log when files are rotated/moved/truncated ([#33237](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33237))
+- (Contrib) `stanza`: Add monitoring metrics for open and harvested files in fileconsumer ([#31256](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/31256))
+- (Contrib) `awss3receiver`: Uses obsreport to report metrics for the AWS S3 Receiver. ([#30750](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/30750))
+- (Contrib) `awsxrayexporter`: AWS X-Ray exporter to make local root spans a segment for internal/service spans and subsegment + segment for client/producer/consumer spans. ([#33000](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33000))
```
[the remark this answers is not in the document]
Comment (samiura) [reply CHANGELOG.md]: removed

File: CHANGELOG.md [hunk 11]
```diff
@@ -2,9 +2,39 @@
...
+
+
+### 🧰 Bug fixes 🧰
+
+- (Splunk) `discovery`: Fix crashing collector if discovered mongodb isn't reachable in Kubernetes ([#4911](https://github.com/signalfx/splunk-otel-collector/pull/4911))
+- (Core) `batchprocessor`: ensure attributes are set on cardinality metadata metric [#9674](https://github.com/open-telemetry/opentelemetry-collector/pull/9674)
+- (Core) `batchprocessor`: Fixing processor_batch_metadata_cardinality which was broken in v0.101.0 [#10231](https://github.com/open-telemetry/opentelemetry-collector/pull/10231)
+- (Core) `batchprocessor`: respect telemetry level for all metrics [#10234](https://github.com/open-telemetry/opentelemetry-collector/pull/10234)
+- (Core) `exporterhelper`: Fix potential deadlocks in BatcherSender shutdown [#10255](https://github.com/open-telemetry/opentelemetry-collector/pull/10255)
+- (Contrib) `podmanreceiver`: add scraper's shutdown method ([#29994](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/29994))
```
Comment (crobert-1) [inline CHANGELOG.md:35]:
```suggestion
```
Not in our distro

**Review by crobert-1 — COMMENTED** [review]
Sorry, I missed another one

File: CHANGELOG.md (hunk 11 above)
Comment (samiura) [reply CHANGELOG.md:35]: thanks. kafkareceiver is not there either.

**Comment by samiura** [comment]
@crobert-1 actually kafkareceiver exists.

File: CHANGELOG.md (hunk 11 above)
Comment (samiura) [reply CHANGELOG.md]: https://github.com/signalfx/splunk-otel-collector/blob/main/internal/components/components.go#L173

Comment (samiura) [reply CHANGELOG.md]: I put the kafkareceiver related changes back.

**Comment by crobert-1** [comment]
_There are still a few open change requests, just FYI_ 👍

File: CHANGELOG.md [hunk 12]
```diff
@@ -21,9 +50,28 @@
...
-### 🧰 Bug fixes 🧰
+### 💡 Enhancements 💡
+
+- (Splunk) `discovery`: Update redis discovery instructions ([#4915](https://github.com/signalfx/splunk-otel-collector/pull/4915))
+- (Splunk) `discovery`: Bring Kafkamatrics receiver into the discovery mode ([#4903](https://github.com/signalfx/splunk-otel-collector/pull/4903))
+- (Core) `mdatagen`: support setting an AttributeSet for async instruments [#9674](https://github.com/open-telemetry/opentelemetry-collector/issues/9674)
+- (Core) `mdatagen`: support using telemetry level in telemetry builder [#10234](https://github.com/open-telemetry/opentelemetry-collector/issues/10234)
+  This allows components to set the minimum level needed for them to produce telemetry. By default, this is set to configtelemetry.LevelBasic. If the telemetry level is below that minimum level, then the noop meter is used for metrics.
+- (Core) `mdatagen`: add support for bucket boundaries for histograms [#10218](https://github.com/open-telemetry/opentelemetry-collector/issues/10218)
+- (Core) `releases`: add documentation in how to verify the image signatures using cosign [#9610](https://github.com/open-telemetry/opentelemetry-collector/issues/9610)
```
Comment (dmitryax) [inline CHANGELOG.md:61]: This one is unrelated to our distro

File: CHANGELOG.md [hunk 13]
```diff
@@ -21,9 +50,28 @@
...
 
-### 🧰 Bug fixes 🧰
+### 💡 Enhancements 💡
+
+- (Splunk) `discovery`: Update redis discovery instructions ([#4915](https://github.com/signalfx/splunk-otel-collector/pull/4915))
+- (Splunk) `discovery`: Bring Kafkamatrics receiver into the discovery mode ([#4903](https://github.com/signalfx/splunk-otel-collector/pull/4903))
+- (Core) `mdatagen`: support setting an AttributeSet for async instruments [#9674](https://github.com/open-telemetry/opentelemetry-collector/issues/9674)
+- (Core) `mdatagen`: support using telemetry level in telemetry builder [#10234](https://github.com/open-telemetry/opentelemetry-collector/issues/10234)
+  This allows components to set the minimum level needed for them to produce telemetry. By default, this is set to configtelemetry.LevelBasic. If the telemetry level is below that minimum level, then the noop meter is used for metrics.
+- (Core) `mdatagen`: add support for bucket boundaries for histograms [#10218](https://github.com/open-telemetry/opentelemetry-collector/issues/10218)
```
Comment (dmitryax) [inline CHANGELOG.md:60]: None of these things are exposed to the end user yet. There is no reason to mention changes to the metadata.yaml interface in our distro.

File: CHANGELOG.md (hunk 4 above)
Comment (dmitryax) [reply CHANGELOG.md:65]: We don't ship the telemetrygen. No reason to mention it

Comment (samiura) [reply CHANGELOG.md:65]: removing it.

File: CHANGELOG.md (hunk 13 above)
Comment (samiura) [reply CHANGELOG.md]: removed.

File: CHANGELOG.md (hunk 12 above)
Comment (samiura) [reply CHANGELOG.md]: removed

**Review by pjanotti — APPROVED** [review]
I do have a Q about blank lines...

File: CHANGELOG.md [hunk 14]
```diff
@@ -2,9 +2,38 @@
...
 ## Unreleased
 
+## v0.102.0
+
 ### 🛑 Breaking changes 🛑
 
 - (Splunk) `receiver/discovery`: Replace `log_record` field with `message` in evaluation statements ([#4583](https://github.com/signalfx/splunk-otel-collector/pull/4583))
+- (Core) `envprovider`: Restricts Environment Variable names.  Environment variable names must now be ASCII only and start with a letter or an underscore, and can only contain underscores, letters, or numbers. ([#9531](https://github.com/open-telemetry/opentelemetry-collector/issues/9531))
+- (Core) `confighttp`: Apply MaxRequestBodySize to the result of a decompressed body [#10289](https://github.com/open-telemetry/opentelemetry-collector/pull/10289)
+
```
[the remark this answers is not in the document]
Comment (samiura) [reply CHANGELOG.md:12]: checking

[the remark this answers is not in the document]
Comment (samiura) [reply CHANGELOG.md]: removed extra spaces, if you look at the contrib/core they have those spaces without issues so I though it makes sense to put blank lines between. At any rate it did not crash ( GH).

At any rate I have removed blank lines.

[the remark this answers is not in the document]
Comment (pjanotti) [reply CHANGELOG.md]: likely auto-generated by chloggen

File: CHANGELOG.md [hunk 15]
```diff
@@ -15,15 +41,24 @@
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>`
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>-2019`
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>-2022`
-
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:18]: Please re-add this line break.  Otherwise there won't be a break with following line.

File: CHANGELOG.md [hunk 16]
```diff
@@ -15,15 +41,24 @@
...
   Starting with this release, the `quay.io/signalfx/splunk-otel-collector:<version>` docker image manifest has been
   updated to support Windows (2019 amd64, 2022 amd64), in addition to Linux (amd64, arm64, ppc64le).
-
```
Comment (jeffreyc-splunk) [inline CHANGELOG.md:21]: Please re-add this line break.  Otherwise there won't be a break with following line.

Comment (jeffreyc-splunk) [reply CHANGELOG.md:21]: See the preview (https://github.com/signalfx/splunk-otel-collector/blob/a0744114d52adae7dc93935c20f0b35574de249f/CHANGELOG.md#-deprecations-) for how the doc will be rendered without the empty lines.

File: CHANGELOG.md (hunk 15 above)
Comment (samiura) [reply CHANGELOG.md]: added back.

File: CHANGELOG.md (hunk 16 above)
Comment (samiura) [reply CHANGELOG.md:21]: I have added back.

**Comment by samiura** [comment]
@jeffreyc-splunk  does it look good now?

https://github.com/signalfx/splunk-otel-collector/blob/8976c6c7d66e13f7dc8c006809964ca394d035bd/CHANGELOG.md

**Comment by jeffreyc-splunk** [comment]
> @jeffreyc-splunk does it look good now?
> 
> https://github.com/signalfx/splunk-otel-collector/blob/8976c6c7d66e13f7dc8c006809964ca394d035bd/CHANGELOG.md

Perfect, thanks!

**Review by dmitryax — APPROVED** [review]
